synapse/cortex.py

```diff
@@ -2048,6 +2060,121 @@ async def syncLayerNodeEdits(self, iden, offs, wait=True):
         async for item in layr.syncNodeEdits(offs, wait=wait):
             yield item
 
+    async def syncNodeFilteredEdits(self, offs, matchdef, wait=True):
+        '''
+        Yield (offs, layriden, (buid, form, individual edits) tuples from the nodeedit logs of all layers starting from
```

Comment:
This seems incorrect vs line 2091 below?

Reply:
OK.